tests/semanticdb/metac.expect

```diff
@@ -281,19 +290,19 @@ Symbols => 13 entries
 Occurrences => 32 entries
 
 Symbols:
-example/Anonymous# => class Anonymous
-example/Anonymous#Foo# => trait Foo
-example/Anonymous#Foo#`<init>`(). => primary ctor <init>
-example/Anonymous#`<init>`(). => primary ctor <init>
-example/Anonymous#foo. => val method foo
-example/Anonymous#locally(). => method locally
-example/Anonymous#locally().(x) => param x
-example/Anonymous#locally().[A] => typeparam A
-example/Anonymous#m1(). => method m1
-example/Anonymous#m1().[T] => typeparam T
-example/Anonymous#m2(). => method m2
-local0 => val local x
-local1 => final class $anon
+example/Anonymous# => class Anonymous extends Object { self: Anonymous & Anonymous => +6 decls }
+example/Anonymous#Foo# => trait Foo extends Object { self: Foo => +1 decls }
+example/Anonymous#Foo#`<init>`(). => primary ctor <init> (): Foo
+example/Anonymous#`<init>`(). => primary ctor <init> (): Anonymous
+example/Anonymous#foo. => val method foo Foo
+example/Anonymous#locally(). => method locally [typeparam A ](param x: A): A
+example/Anonymous#locally().(x) => param x: A
+example/Anonymous#locally().[A] => typeparam A 
+example/Anonymous#m1(). => method m1 [] => Nothing
```

Comment:
curiously empty type parameters here

Reply:
Fixed in https://github.com/lampepfl/dotty/pull/12885/commits/a8a137c36977125b3d65e2c2289268961ce2db14